SPECS/python3-cassandra-driver/python3-cassandra-driver.spec

```diff
@@ -0,0 +1,84 @@
+Summary:        A modern, feature-rich and highly-tunable Python client library for Apache Cassandra (2.1+)
+Name:           python3-cassandra-driver
+Version:        3.24.0
+Release:        5%{?dist}
+Url:            https://github.com/datastax/python-driver#datastax-python-driver-for-apache-cassandra
+License:        ASL 2.0
+Group:          Development/Languages/Python
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Source0:        https://github.com/datastax/python-driver/archive/refs/tags/%{version}.tar.gz#/cassandra-driver-%{version}.tar.gz
+BuildRequires:  python3
+BuildRequires:  python3-Cython
+BuildRequires:  python3-libs
+BuildRequires:  python3-devel
+BuildRequires:  python3-setuptools
+BuildRequires:  python3-pip
+BuildRequires:  libev-devel
+BuildRequires:  libev
+%if %{with_check}
+BuildRequires:  python3-pytest
+BuildRequires:  python3-packaging
+BuildRequires:  openssl-devel
+BuildRequires:  curl-devel
+BuildRequires:  iana-etc
+%endif
+Requires:       libev
+Requires:       python3
+Requires:       python3-libs
+Requires:       python3-six
+Requires:       python3-geomet < 0.3
+
+%description
+A modern, feature-rich and highly-tunable Python client library for Apache Cassandra (2.1+)
+using exclusively Cassandra's binary protocol and Cassandra Query Language v3. The driver
+supports Python 2.7, 3.3, 3.4, 3.5, and 3.6.
+
+%prep
+%autosetup -n python-driver-%{version}
+
+%build
+%{py3_build "--no-cython"}
+
+%install
+%py3_install
+
+%check
+pip3 install nose scales mock ccm unittest2 pytz sure pure-sasl twisted gevent eventlet packaging Netbase
```

Comment:
Is there a test requirements file we can reference instead of doing this?

Reply:
Yes, added the test-requirements.txt to list all dependencies to be installed by pip.